### PR TITLE
[WFLY-3839] No cluster node manager found for node XY during server rest...

### PR DESCRIPTION
**Pull request by TomasHofman**
...art

- error message changed to debug, since the situation is expected to happen

Issue: https://issues.jboss.org/browse/WFLY-3839

### CODE REVIEW

File: src/main/java/org/jboss/ejb/client/ClusterContext.java [hunk 1]
```diff
@@ -410,7 +410,7 @@ public Void call() throws Exception {
             if (clusterNodeManager == null) {
                 // we don't have a cluster node manager which could create the EJB receiver, for this
                 // node name
-                logger.error("Cannot create EJBReceiver since no cluster node manager found for node "
+                logger.debug("Cannot create EJBReceiver since no cluster node manager found for node "
```
Comment (wolfc) [inline src/main/java/org/jboss/ejb/client/ClusterContext.java]: Use the f method instead of StringBuilder concats.

**Comment by dmlloyd** [comment]
Can you squash these two into one commit please?

**Comment by dmlloyd** [comment]
Actually nevermind, I'll do by hand.

**Comment by dmlloyd** [comment]
Merged manually.

**Pull request closed without merging**